types/setting.go

```diff
@@ -1112,6 +1115,16 @@ var (
 		ReadOnly:    false,
 		Default:     "5",
 	}
+
+	SettingDefinitionLogLevel = SettingDefinition{
+		DisplayName: "Log Level",
+		Description: "The log level 0 (Panic) ~ 6 (Trace) used in longhorn manager. By default 3 (Warning).",
```

Comment:
Let's use level constant instead of number. You can use `ParseLevel` to achieve that.

Reply:
Fixed, thanks